<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Posicion x:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="posi_x"/>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Radio: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="radio_line"/>
          </item>
         </layout>
        </item>
        <item row="10" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="inicio">
            <property name="text">
             <string>Inicio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pausa">
            <property name="text">
             <string>Pausa</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Posicion y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="posi_y"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Velocidad en x: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="velocidad_x"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="boton_cantidad_planetas_2">
          <property name="text">
           <string>Cantidad de planetas</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Velocidad en y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="velocidad_y"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Cantidad de planetas: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="can_planetas"/>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Masa: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="masa"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>INGRESAR</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="ingresar_datos">
          <property name="text">
           <string>Ingresar datos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
